stdlib/3/email/message.pyi

```diff
@@ -34,9 +34,9 @@ class Message:
     def get_charset(self) -> _CharsetType: ...
     def __len__(self) -> int: ...
     def __contains__(self, name: str) -> bool: ...
-    def __getitem__(self, name: str) -> Optional[_HeaderType]: ...
-    def __setitem__(self, name: str, val: _HeaderType) -> None: ...
-    def __delitem__(self, name: str) -> None: ...
+    def __getitem__(self, name: str) -> Any: ...
+    def __setitem__(self, name: str, val: _HeaderType) -> Any: ...
```

Comment:
Why did you change the return types? I think the argument type is the one that needs changing.

Reply:
@JelleZijlstra, would it make sense to have the argument type defined like this:
`_HeaderType = Union[str, Header, ContentTransferEncodingHeader, ContentTypeHeader, UniqueDateHeader, SingleAddressHeader, AddressHeader, UniqueAddressHeader, UniqueSingleAddressHeader, MIMEVersionHeader, UniqueUnstructuredHeader, DateHeader, ContentDispositionHeader]`?

Reply:
That's way too big a union to be a good idea.

Reply:
Right on, what's the best approach then?

Reply:
Hi @JelleZijlstra.  
Whilst going over the pending tasks, I stumbled upon this.  
What'd be the best approach here?
CC: @srittau.

Reply:
I think the best way forward is to change `_HeaderType` to `Any` due to the complicated type situation and the fact it is used in return types. The `__getitem__()` should return `_HeaderType`. `__setitem__()` and `__delitem__()` should continue to return `None`, though.

Reply:
Thank you, fixed!